apps/qtmozembed/declarativewebpage.cpp

```diff
@@ -86,11 +85,10 @@ DeclarativeWebPage::DeclarativeWebPage(QObject *parent)
             this, &DeclarativeWebPage::onRecvAsyncMessage);
     connect(&m_grabWritter, &QFutureWatcher<QString>::finished, this, &DeclarativeWebPage::grabWritten);
     connect(this, &DeclarativeWebPage::urlChanged, this, &DeclarativeWebPage::onUrlChanged);
-    connect(this, &QOpenGLWebPage::contentHeightChanged, this, &DeclarativeWebPage::updateViewMargins);
+    connect(this, &QOpenGLWebPage::virtualKeyboardHeightChanged, this, &DeclarativeWebPage::updateViewMargins);
```

Comment:
As far as I can tell (using this PR and https://git.sailfishos.org/mer-core/qtmozembed/merge_requests/74 ) the `QOpenGLWebPage::virtualKeyboardHeight` is never changed, and so this signal never triggers. Can this connection be removed?

Reply:
Really? See QOpenGLWebPage::setVirtualKeyboardHeight(int height)

Reply:
Ah, now I understand what you're saying... Please this on top of https://github.com/sailfishos/sailfish-browser/pull/741